src/features/MoviesBrowser/ListPage/MovieTile/styled.js

```diff
@@ -0,0 +1,67 @@
+import styled, { css } from "styled-components";
+
+export const StyledMovieTile = styled.div`
+    background: ${({ theme }) => theme.colors.tileBackground};
+    padding: 40px;
+    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+    display: grid;
+    grid-template-columns: auto 1fr;
+    grid-gap: 40px;
+    border-radius: 5px;
+
+    ${({ horizontal }) =>
+    horizontal &&
+    css`
+      max-width: 100%;
+      display: grid;
+      grid-template-columns: 312px 1fr;
+      grid-gap: 40px;
+      margin-top: 20px;
+      margin-bottom: 49px;
+    `}
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.mobile}) {
+    margin: 0px;
+    width: 100%;
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns: 1fr 1fr;
+    align-items: center;
+  }
+`;
+
+
+export const Title = styled.header`
+    font-weight: 600;
+    font-size: 36px;
+    margin-bottom: 24px;
+`;
+
+export const Subtitle = styled.div`
+    font-size: 22px;
+`;
+
+export const Content = styled.div``;
+
+export const Tags = styled.ul`
```

Comment:
Nie robił bym tego jako lista ale jak już jest to trzeba zrobić aby się wyświetlało w odpowiedni sposób

Reply:
Zobaczę co da się z tym zrobić :P